consensus/mempool_test.go

```diff
@@ -256,3 +256,8 @@ func (app *CounterApplication) Commit() abci.ResponseCommit {
 	binary.BigEndian.PutUint64(hash, uint64(app.txCount))
 	return abci.ResponseCommit{Data: hash}
 }
+
+func (app *CounterApplication) PrepareProposal(
+	req abci.RequestPrepareProposal) abci.ResponsePrepareProposal {
+	return abci.ResponsePrepareProposal{BlockData: req.BlockData}
```

Comment:
[This comment](https://github.com/celestiaorg/celestia-core/pull/898/files#r1037958275) also applies here